packages/react-native-worklets/Common/cpp/worklets/NativeModules/WorkletsModuleProxy.cpp

```diff
@@ -120,6 +120,13 @@ jsi::Value WorkletsModuleProxy::createWorkletRuntime(
   return jsi::Object::createFromHostObject(rt, workletRuntime);
 }
 
+void WorkletsModuleProxy::terminateWorkletRuntime(
+    jsi::Runtime &rt,
+    const jsi::Value &workletRuntimeValue) {
+  auto workletRuntime = extractWorkletRuntime(rt, workletRuntimeValue);
```

Comment:
Can we use `const auto &` here?
```suggestion
  const auto &workletRuntime = extractWorkletRuntime(rt, workletRuntimeValue);
```